pkg/databaseclaim/databaseclaim.go

```diff
@@ -2722,7 +2763,7 @@ func (r *DatabaseClaimReconciler) createSecret(ctx context.Context, dbClaim *v1.
 			"sslmode":        []byte(connInfo.SSLMode),
 		},
 	}
-	logr.Info("creating connection info secret", "secret", secret.Name, "namespace", secret.Namespace)
+	logr.Info("creating connection info SECRET: "+secret.Name, "secret", secret.Name, "namespace", secret.Namespace)
```

Comment:
perhaps this, the name is duplicated in this change. Also it's easier to search for single words than when separated by spaces
```suggestion
	logr.Info("creating_connection_info", "secret", fmt.Sprintf("%s/%s", secret.Namespace, secret.Name))
```